test/e2e/benchmark/benchmark.go

```diff
@@ -20,14 +25,30 @@ type BenchmarkTest struct {
 }
 
 func NewBenchmarkTest(name string, manifest *Manifest) (*BenchmarkTest, error) {
-	// create a new testnet
-	testNet, err := testnet.New(context.Background(), name, seed,
-		testnet.GetGrafanaInfoFromEnvVar(), manifest.ChainID,
-		manifest.GetGenesisModifiers()...)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	identifier := fmt.Sprintf("%s_%s", name, time.Now().Format(timeFormat))
+	kn, err := knuu.New(ctx, knuu.Options{
+		Scope:        identifier,
+		ProxyEnabled: true,
+	})
 	if err != nil {
 		return nil, err
 	}
 
+	// context.Background() is used to allow the stopSignal to be functioning even after this function returns
+	kn.HandleStopSignal(context.Background())
+
+	log.Printf("Knuu initialized with scope %s", kn.Scope)
```

Comment:
[nit] I would move this log statement immediately beneath the Knuu initialization command (and error handling):

```go
	kn, err := knuu.New(ctx, knuu.Options{
		Scope:        identifier,
		ProxyEnabled: true,
	})
	if err != nil {
		return nil, err
	}
	log.Printf("Knuu initialized with scope %s", kn.Scope)
```